<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QLineEdit" name="textbox">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>20</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sum">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equal">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_7">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_8">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_10">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decimal">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="percent">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiply">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divide">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>190</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cos">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tan">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cot">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>cot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AC">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DEL">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>DEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sqrt">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
